flink/src/test/java/org/apache/iceberg/flink/sink/TestTaskWriters.java

```diff
@@ -235,11 +252,115 @@ public void testRandomData() throws IOException {
     }
   }
 
+  @Test
+  public void testWriteEqualityDelete() throws IOException {
+    if (format == FileFormat.ORC) {
```

Comment:
Can you use `Assume.assumeTrue`? That way it doesn't look like ORC is passing. It shows up as skipped.

Reply:
Yes, make sense !